cpp/ql/src/semmle/code/cpp/exprs/Expr.qll

```diff
@@ -1268,3 +1268,32 @@ class SpaceshipExpr extends BinaryOperation, @spaceshipexpr {
 
   override string getOperator() { result = "<=>" }
 }
+
+/**
+ * A C/C++ co_await expression
+ * ```
```

Comment:
```suggestion
 * ```cpp
```

Reply:
Thanks for the suggestion, but I think that for consistency, if we're going to do this then we should do it for all QL examples at once, in a separate PR, rather than just doing it for these new ones here.

Reply:
I can understand your thought, but at the same time doing that for all QL examples would be a huge amount of work.
I know that only doing this for new code leads to inconsistency, but if you never start you just have more work for when you also do the other examples.
Anyway, feel free to ignore my suggestions :)